src/webob/cookies.py

```diff
@@ -239,10 +240,13 @@ def serialize_cookie_date(v):
         v = timedelta(seconds=v)
 
     if isinstance(v, timedelta):
-        v = datetime.utcnow() + v
+        v = datetime.now(tz=zoneinfo.ZoneInfo("UTC")) + v
 
-    if isinstance(v, (datetime, date)):
+    if isinstance(v, datetime):
+        v = v.astimezone(zoneinfo.ZoneInfo("UTC")).timetuple()
+    elif isinstance(v, date):
         v = v.timetuple()
```

Comment:
The function can receive `bytes`, `str`, `int` but it doesn’t have an else branch to catch any other unexpected type. If for some reason a datetime is passed in then we’ll want to make sure it’s in UTC, hence the adjustment. Also, unless a date object is passed in there shouldn’t be a need to handle those but cookies should always contain a date & time AFAIK.